services/email/templates/customer/plain/delivery.mus

```diff
@@ -26,3 +22,4 @@ If you're unsure an email is from UKVI:
  * do not reply to it or click any links
 
 {{/data}}
+
```

Comment:
Did you mean to add an extra line?

Reply:
Ah no, think my linting thing told me to. Will remove that.

Reply:
👍